linux/keyman-config/keyman_config/install_kmp.py

```diff
@@ -234,6 +239,13 @@ def install_kmp_user(inputfile, online=False):
 			elif ftype == KMFileTypes.KM_SOURCE:
 				#TODO for the moment just leave it for ibus-kmfl to ignore if it doesn't load
 				pass
+			elif ftype == KMFileTypes.KM_KMX:
+				# Sanitize keyboard filename if not lower case
+				kmx_id = re.sub('.kmx', '', f['name'])
```

Comment:
that also works, though os.path.splitext was the way I had been getting the filename part without extension
e.g. `packageID, ext = os.path.splitext(os.path.basename(inputfile))`